lib/components/app/app-menu-item.tsx

```diff
@@ -0,0 +1,101 @@
+import { ChevronDown } from '@styled-icons/fa-solid/ChevronDown'
+import { ChevronUp } from '@styled-icons/fa-solid/ChevronUp'
+import AnimateHeight from 'react-animate-height'
+import React, { Component, HTMLAttributes, KeyboardEvent } from 'react'
+
+interface Props extends HTMLAttributes<HTMLElement> {
+  href?: string
+  icon?: JSX.Element
+  isDropdown?: boolean
+  onClick?: () => void
+  subItems?: unknown[]
+  text: JSX.Element | string
+}
+
+interface State {
+  isExpanded: boolean
+}
+
+/**
+ * Helper method to find the element within the app menu at the given offset
+ * (e.g. previous or next) relative to the specified element.
+ * The query is limited to the app menu so that arrow navigation is contained within
+ * (tab navigation is not restricted).
+ */
+function getEntryRelativeTo(element: EventTarget, offset: 1 | -1): HTMLElement {
+  const entries = Array.from(
+    document.querySelectorAll('.app-menu a, .app-menu button')
```

Comment:
This isn't the best but given the fact that we don't have time to rebuild the menu from scratch happy to let this one pass. We should maybe add a TODO to rebuild the entire menu? I feel it's very bloated -- I don't think we need the library it uses and some of the styling is overkill as well I feel.

Reply:
Which library? `AnimateHeight`?

Reply:
`react-sliding-pane`